modules/db/src/main/java/org/mule/module/db/internal/domain/connection/DefaultDbConnection.java

```diff
@@ -337,7 +342,7 @@ public Struct createStruct(String typeName, Object[] attributes) throws SQLExcep
         }
         catch (SQLException e)
         {
-            logger.warn("Unable to resolve lobs. Proceeding with original attributes.");
+            logger.warn("Unable to resolve lobs. Proceeding with original attributes.", e);
```

Comment:
this will log the full stack trace, only the message of the exception is of interest